template.js

```diff
@@ -76,10 +83,46 @@ if (eventName === 'unknown') {
   return data.gtmOnFailure();
 }
 
+// Return the existing 'BillyPix' global method if available
+let BillyPix = copyFromWindow(billyFunctionName);
+
+function loadBackupBilly (BillyPixBackup) {
+  // Initialize BillyPix so the Tracking ID is set on the web page
+  BillyPixBackup('init', billyPixId);
+  
+  // Debug to see if correct
+  if (data.isDebug){
+    log('Backup: Successfully initialized the ' + billyFunctionName + ' for ID: ' + billyPixId);
+  }
+}
+
+function loadLibraryIfNotAvailable() {
+  // Function to ensure BillyPix is defined and properly queues commands
+  const BillyPixBackup = createArgumentsQueue(billyFunctionName, billyFunctionName + '.queue');
+  
+  // Setup BillyPix with the current timestamp
+  setInWindow(billyFunctionName + '.t', getTimestamp(), false);
+
+  // Generate the script URL with cache busting
+  const secondsBuste = 30*1000;
+  const epochRounded = secondsBuste * Math.ceil(getTimestamp() / secondsBuste);
+  const scriptUrl = cdnEndpoint + '?t=' + epochRounded + '&v=0.2.0';
```

Comment:
What the version does here?
Do we wanna make this a variable and re use it in other URLs ?

Reply:
Yeah so this is the way we normally load the bundle, but you are right the version we don't actually use. Only the `t` part is useful to buste the cache automatically as the url changes